compiler/src/typed/typemod.re

```diff
@@ -826,13 +826,8 @@ let type_module = (~toplevel=false, funct_body, anchor, env, sstr /*scope*/) =>
 
 let type_module = type_module(false, None);
 
-let implicit_modules: ref(list((string, string, bool))) = (
-  ref([
-    ("Pervasives", "pervasives", true),
-    ("Malloc", "runtime/malloc", false),
-  ]):
-    ref(list((string, string, bool)))
-);
+let implicit_modules: ref(list((string, string, bool))) =
+  ref([("Pervasives", "pervasives", true), ("GC", "runtime/gc", false)]);
```

Comment:
This makes me think: doesn't this mean that all modules, even in `Normal` mode, can just casually call `GC` functions? I suppose it's fine, but it gives me an odd gut reaction.

Reply:
Oh, no they can't. That `false` at the end instructs the loader to not put any of the values in the environment.